vizro-core/hatch.toml

```diff
@@ -111,7 +111,9 @@ VIZRO_LOG_LEVEL = "DEBUG"
 extra-dependencies = [
   "pydantic==1.10.16",
   "dash==2.17.1",
-  "plotly==5.12.0"
+  "plotly==5.12.0",
+  "pandas==2.0.0",
+  "numpy==1.23.0"  # Need numpy<2 to work with pandas==2.0.0. See https://stackoverflow.com/questions/78634235/.
```

Comment:
It was a somewhat arbitrary choice but I deliberately went for something that's relatively old rather than tight against the `numpy<2` bound. 1.26.4 is from February 2024 while 1.23.0 is from June 2022. With these lower bounds tests we should be testing against something that's old rather than recent.